webrender/src/prim_store.rs

```diff
@@ -718,10 +718,12 @@ impl PrimitiveStore {
 
     pub fn resolve_primitives(&mut self,
                               resource_cache: &ResourceCache,
-                              device_pixel_ratio: f32) -> Vec<DeferredResolve> {
+                              device_pixel_ratio: f32,
+                              layer_store: &Vec<StackingContext>,
+                              auxiliary_lists_map: &AuxiliaryListsMap) -> Vec<DeferredResolve> {
```

Comment:
Ideally, I'd like to see the auxiliary lists accessed from a single place, like `cull_layers`. Although, it doesn't seem to have any other downsides here than the code complexity, so we can have it for now and then tentatively look into streamlining the use of the aux lists.